### ISSUE TITLE
VST not showing up in Ableton

### ISSUE THREAD

**Issue by daintym0sh** [diracdeltas/vstSpleeter#8]
I followed the instructions below, but the VST does not show up in my list of VST3's within Ableton. The asset contents have been added to the VST3 custom scan path, though it does not show up.

https://github.com/diracdeltas/vstSpleeter/releases/tag/v0.2.1

**Comment by paulkrupin** [comment]
Yesterday I tried to add this plugin to Ableton 11 on Mac with Catalina. 10th version showing it and 11th isn't.

**Comment by paulkrupin** [comment]
Today there was a chance to try VST on Windows 10 (2004) with Ableton Live 11. The plug-in appeared in the library and working just as expected.

**Comment by kaalachandra** [comment]
it does not show in ableton 11 on windows.
it sees it when it scans but then it is not available

Please help

**Comment by diracdeltas** [comment]
@kaalachandra are you on macos? if so maybe https://github.com/diracdeltas/vstSpleeter/issues/4#issuecomment-634195448 would solve it

**Comment by kaalachandra** [comment]
No, sorry. I'm on Mac, I checked the discussion and no helpIl 28 mag 2022 19:38, yan ***@***.***> ha scritto:
@kaalachandra are you on macos? if so maybe #4 (comment) would solve it

**Comment by kaalachandra** [comment]
I mean, on windows!Il 28 mag 2022 19:45, alessandro bortolozzi ***@***.***> ha scritto:

**Comment by diracdeltas** [comment]
@kaalachandra sorry i don't have a windows machine anymore to test on :(. if your version of ableton supports Max For Live, it may be easier if you just use this max device I recently made: https://github.com/diracdeltas/demucs4max#quickstart. the quality of separation is much better but the latency is very high.

**Comment by princessdasha** [comment]
@diracdeltas i'm having the same issue as @kaalachandra but on Mac (Monterey 12.4 / Apple M1 Pro / Live 11 run with Rosetta). I've tried the comment #4  workaround in all the variations I can think of. Ableton will scan the plugin and the total plugin count will go up by 1 after I remove the quarantine attribute from the plugin and/or .zip file, but then it won't display in the plugins menu. It also doesn't seem to appear in Live 10 on the same machine, or in Reaper. Is there anything else I should try? Any tests I could do? Any further info that might help with problem solving? I loved this device so much on my older mac and for a variety of reasons it suites my musical purposes in ways the demucs plugin doesn't - I would be thrilled to help with problem solving if I could!

**Comment by diracdeltas** [comment]
@princessdasha does it run at all if you start ableton NOT in rosetta?

**Comment by daintym0sh** [comment]
Another option is to run with Docker. The spleeter wiki has a guide on how to do this, provided you have Docker installed:
https://github.com/deezer/spleeter/wiki/2.-Getting-started#run-container

This has been my preferred way of using spleeter because Docker is the only dependency required.

**Comment by princessdasha** [comment]
@diracdeltas no luck running not in rosetta. ableton does the same thing: will scan the plugin, but won't show it or even an empty 'Azuki' folder. When I run xattr in terminal with the plugin, it looks like the quarantine attribute is gone (the only attributes seem to be 'com.apple.lastuseddate#PS' and 'com.apple.macl')

@daintym0sh thanks for this! my ability to parse coding language isn't great, but from what i'm seeing it looks like the docker solution still requires you to supply a full file of the song you want to separate into stems, and then wait while the program does the separation? For the application I am using stem separation for, I need the ability (even with fairly high latency) to process a live input (in my case a live feed from an fm radio). So far vstSpleeter is the only way I've found of doing this...

**Comment by daintym0sh** [comment]
Ah I see, you're right. In that case VST makes sense, so you can have real-time processing. I haven't revisited this in quite some time. I'll take a look. Hope you can find a solution.

**Comment by princessdasha** [comment]
@daintym0sh thanks so much!
